TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -90,9 +91,7 @@ class RouteOptionsViewController: UIViewController {
 
         // assume user wants to find routes that leave at current time and set datepicker accordingly
         searchTime = Date()
-        if let searchTime = searchTime {
-            routeSelection.setDatepickerTitle(withDate: searchTime, withSearchTimeType: searchTimeType)
-        }
+        routeSelection.setDatepickerTitle(withDate: searchTime!, withSearchTimeType: searchTimeType)
```

Comment:
can we unwrap `searchTime`. I know in this case searchTime isn't going to be nil but its safer to unwrap in case someone ends up changing this code in the future